aviary/interface/methods_for_level2.py

```diff
@@ -1724,14 +2122,25 @@ def alternate_mission(self, run_mission=True,
         prob_alternate.set_initial_guesses()
         if run_mission:
             prob_alternate.run_aviary_problem(
-                record_filename='alternate_problem_history.db')
+                record_filename='alternate_problem_history.db'
+            )
         return prob_alternate
 
-    def fallout_mission(self, run_mission=True,
-                        json_filename='sizing_problem.json',
-                        num_first=None, num_business=None, num_tourist=None, num_pax=None,
-                        wing_cargo=None, misc_cargo=None, cargo_mass=None, mission_mass=None,
-                        phase_info=None, verbosity=Verbosity.BRIEF):
+    def fallout_mission(
```

Comment:
I really don't like this expanded format, it doesn't add anything and takes up waay too many lines of code.